### PR TITLE
l10n: uk: update core translation

### PR DESCRIPTION
**Pull request by arkid15r**

### CODE REVIEW

File: po/uk.po [hunk 1]
```diff
@@ -1387,15 +1395,21 @@ msgid ""
 "    git push --set-upstream %s %s\n"
 "%s"
 msgstr ""
+"Поточна гілка %s не має гілки у першоджрельному сховищі.\n"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
"Поточна гілка %s не має гілки у першоджeрельному сховищі.\n"
```

File: po/uk.po [hunk 2]
```diff
@@ -1429,17 +1454,25 @@ msgid ""
 "(e.g., 'git pull ...') before pushing again.\n"
 "See the 'Note about fast-forwards' in 'git push --help' for details."
 msgstr ""
+"Оновлення було відхилено, оскільки у віддаленому сховищі міститься робота, якої ви\n"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
"Оновлення було відхилено, оскільки у віддаленому сховищі міститься робота, якої\n"
```

File: po/uk.po [hunk 3]
```diff
@@ -1429,17 +1454,25 @@ msgid ""
...
+"немає локально. Зазвичай це стається, коли інше сховище надсилає\n"
+"до того самого посилання. Можливо, вам слід спочатку інтегрувати віддалені зміни\n"
+"(наприклад, через “git pull …”) перед повторним надсаланням.\n"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
"(наприклад, через “git pull …”) перед повторним надсиланням.\n"
```

File: po/uk.po [hunk 4]
```diff
@@ -1448,11 +1481,15 @@ msgid ""
 "to integrate those changes locally (e.g., 'git pull ...')\n"
 "before forcing an update.\n"
 msgstr ""
+"Оновлення було відхилено, оскільки верхівка гілки віддаленого відстеження\n"
+"гілки було оновлено з моменту останнього стягування. Можливо, ви захочете\n"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
"гілки була оновлена з моменту останнього стягування. Можливо, ви захочете\n"
```

**Review by kasya — COMMENTED** [review]
LGTM!
Let's do it! 💪🏼

File: po/uk.po (hunk 2 above)
Comment (arkid15r) [reply po/uk.po]: 👍

**Pull request merged**